assets/penalty_types.yml

```diff
@@ -102,7 +102,7 @@ allowed_transactions:
       true
     NN:
       true
-    NO:
+    'NO':
```

Comment:
Should this have quotes as none of the others do?

Reply:
I could quote the others too

This one is quoted and I am hoping that Sonar would recognise this literal is a token, rather than a boolean truthy value

Reply:
Now this has been merged to we know if by putting quotes on this particular penalty type still works ok?